webapp/src/index.js

```diff
@@ -19,6 +19,10 @@ export default class Plugin {
             (channelId) => {
                 store.dispatch(openMeetingSettingsModal(channelId));
             });
+
+        registry.registerPostDropdownMenuAction('Re-queue', (itemId) => {
```

Comment:
We possibly want to use an identifier in `post.props` to signal that it's an agenda post. But that wouldn't work with manually created agenda items.

Reply:
0/5, for now we could ensure we only show `Re-queue` option if a post starts with an `#`. Your thoughts? @mickmister thanks.

Reply:
> 0/5, for now we could ensure we only show `Re-queue` option if a post starts with an `#`

I think this is a little too broad of a claim. Two reasons come to mind:

1. Posts can start with `#` because they are using a header, and not a hash tag. For example,  `# I'm a header` renders as:

# I'm a header

2. Hashtags don't imply that it's an agenda hashtag.

I think we should go with the `post.props` strategy. If someone wants to use the post menu to re-queue an item they created manually, too bad. cc @hanzei

Another option going along with this same strategy is for the server-side of the plugin to apply the `post.props` mutation to the manually created agenda items during the `MessageWillBePosted` and `MessageWillBeUpdated` hooks.

Reply:
:+1: for using `post.props`

Reply:
Thanks @mickmister  @hanzei , nice approach.
Should I implement it in this PR or create a separate issue?

Reply:
I'm thinking we should add it to this PR. @hanzei thoughts?

Reply:
@sanjaydemansol Note that the PR should be using `post.props` to determine if the post is created by the plugin